### ISSUE TITLE
feat: Expose the template render engine to be used programatically

### ISSUE THREAD

**Issue by erickzanardo** [felangel/mason#150]
**Description**

When used as a package dependency, there is no way (at least I didn't find any 😅 ) to use the template engine of mason programatically. Unless we import source files from mason (like I did [here](https://github.com/erickzanardo/quary/blob/main/lib/src/repository/snippet_repository.dart#L6))

Would be cool if mason could expose that file, so we could also use its cool template engine programatically.

**Comment by felangel** [comment]
Hi @erickzanardo 👋 
Thanks for opening an issue!

If you just want to use the mustache template engine you can use [package:mustache_template](https://pub.dev/packages/mustache_template) directly rather than using mason.

```dart
import 'package:mustache_template/mustache_template.dart';

final template = Template('Hello {{ name }}');
final rendered = template.renderString({'name': 'Dash'}); // 'Hello Dash'
```

Let me know if that helps 👍

**Comment by felangel** [comment]
This is now available in [mason v0.1.0-dev.2](https://pub.dev/packages/mason/versions/0.1.0-dev.2):

```dart
final output = 'Hello {{name}}!'.render({'name': 'Dash'});
print(output); // 'Hello Dash!
```
